cli/util/upload_bin.go

```diff
@@ -235,7 +240,7 @@ func fetchLatestMissingBlockIndex(ctx context.Context, p *pool.Pool, containerID
 				continue
 			}
 			if emptyBatchFound || (batch == numBatches && i == len(results)-1) {
-				return results[i].startIndex, nil
+				return results[i].startIndex / uploadBatchSize * uploadBatchSize, nil
```

Comment:
Separate fix?

Reply:
Not really, as introducing a separate constant `uploadBatchSize` can lead to misalignment between searching and unloading batches. Returning results[i].startIndex - we can miss holes. Previously `startIndex` was always aligned with the start index of the uploading batch as we had the same constant for searching and uploading.